src/components/Header/Header.tsx

```diff
@@ -0,0 +1,32 @@
+import "./Header.scss";
+import { Link } from "react-router-dom";
+
+type HeaderProps = {
+  title: string;
+  subtitle?: string;
+  profileImage?: string;
+};
+
+const Header = ({ title, subtitle, profileImage }: HeaderProps) => {
+  return (
+    <header className="header">
+      <div className="header__image-div">
+        <Link to="/profile">
+          <img
+            className="header__image-div header__image-div--image"
+            src={profileImage}
+          />
+        </Link>
+      </div>
+
+      <div className="header__title-div">
+        <h4 className="header__title-div header__title-div--title">{title}</h4>
```

Comment:
I don't think this needs to be a prop as it is WAY OF THE VIKING on all of the pages